inc/event.class.php

```diff
@@ -432,3 +440,8 @@ static function getCountLogin() {
    }
 
 }
+
+// For compatibility
+if (!class_exists('Event', false)) {
```

Comment:
As far as I understand, this will be compatible only if the pecl-event is not present?

What can use the old name? Plugins?